components/03-modules/newsletter/_newsletter.scss

```diff
@@ -0,0 +1,90 @@
+$newsletter__font-size                : $font-size-base !default;
+$newsletter__font-size--agreements    : $font-size-extra-small !default;
+$newsletter__line-height              : 1.2 !default;
+$newsletter__input-background         : #717171 !default;
+$newsletter__input-placholder-opacity : 0.7 !default;
+$newsletter__heading-color            : #ffffff;
+$newsletter__controls-height          : 32px;
+
+.newsletter {
+    &__heading {
+        line-height: $newsletter__line-height;
+        margin-right: 20px;
+        color: $newsletter__heading-color;
+        font-family: $font-family-sans-serif;
+        height: 18px;
+        font-size: $newsletter__font-size;
+        font-weight: bold;
```

Comment:
please use `$font-weight-bold ` variable

Reply:
so it can change? like 4font-weight-bold we would use: normal? than I think it we shoudl change $font-weight-bold to $font-weight-important  and use font-weight: $font-weight:important;

Reply:
IMO It does not make sense change to `$font-weight-bold: 'normal';` but we have a variable so that so we should use that. If you have any doubts please ask on alpaca room unless just change it.

Reply:
What I was asking if we will change this variable value and if yes than changing to values like for "normal",, "light", make no sens. But I know how it is supposed to be used and what values can be as alternative.

Reply:
Changed